src/Core/src/Hosting/MauiAppBuilder.cs

```diff
@@ -104,6 +109,12 @@ ILoggingBuilder InitializeLogging()
 			}
 		}
 
+		IDictionary<object, object> IHostApplicationBuilder.Properties => throw new NotImplementedException();
+
+		IHostEnvironment IHostApplicationBuilder.Environment => throw new NotImplementedException();
+
+		IMetricsBuilder IHostApplicationBuilder.Metrics => throw new NotImplementedException();
```

Comment:
This is interesting and I am not sure what it is used for or if it is relevant to mobile.

> Allows enabling metrics and directing their output.

Reply:
Do you expect metrics/meters in the Maui application to show up in the Aspire Dashboard? If yes, then it would be useful to implement this. See https://github.com/dotnet/runtime/issues/85684 for more information.